src/libraries/illink-sharedframework.proj

```diff
@@ -0,0 +1,82 @@
+<Project Sdk="Microsoft.Build.Traversal">
```

Comment:
You are using the Traversal SDK but I don't see any ProjectReferences in this project. Also I don't understand (yet) why we need an extra project for this instead of a targets file.

Reply:
@ViktorHofer are you recommending to make this a target that is run by the src.proj or some other project instead?  Do you have a recommended project to put this in?

Reply:
Is there are reason why this can't be a target instead? An extra project which is restored and built as part of src.proj is a bit of an overkill. So yes, a target in src.proj should be sufficient.

I was also commenting on the Traversal SDK being used without any projects being referenced in it which is why I was commenting on this.

Reply:
>  a target in src.proj should be sufficient.

Seems reasonable.

> the Traversal SDK being used without any projects being referenced

Probably it wasn't clear which SDK to use that provides basic "build" targets.  In the future, which SDK would you reccomend for this instead?

Reply:
I think for those cases we should use the NoTargets SDK.

Reply:
Thanks. I'll change this to a targets file.